sdk/tests/conformance/textures/misc/00_test_list.txt

```diff
@@ -20,6 +20,7 @@ origin-clean-conformance.html
 tex-image-and-sub-image-2d-with-array-buffer-view.html
 tex-image-and-uniform-binding-bugs.html
 --min-version 1.0.3 tex-image-canvas-corruption.html
+tex-image-svg-image-no-natural-width-and-height.html
```

Comment:
For bookkeeping purposes we'll want to add `--min-version=1.0.4` here. Let me try to add that directly.